resources/js/store/index.js

```diff
@@ -13,12 +13,20 @@ const getters = {
 const actions = {
     add: function ({commit}, todo) {
         commit('ADD', todo);
+    },
+
+    delete: function ({commit}, todo) {
```

Comment:
If this function is only accepting the ID for the todo to remove, change the name of the parameter accordingly